apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/WriteTests.cs

```diff
@@ -284,17 +182,133 @@ public WriteTests(TestDatabaseFixture testFixture, ITestOutputHelper outputHelpe
         }
 
         [Fact]
-        public async Task WriteNanValue()
+        public async Task BadColumnName()
```

Comment:
resharper code cleanup.  Apologies, but this move will make later cleanups have a smaller diff.